aten/src/ATen/native/cuda/ReduceOpsKernel.cu

```diff
@@ -119,9 +126,14 @@ static void mean_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+static void and_kernel_cuda(TensorIterator& iter) {
+  and_kernel_impl(iter);
```

Comment:
Have to do the wrapping here, otherwise it hits the following error on windows:

```
error : On Windows, the enclosing parent function ("and_kernel_cuda") for an extended __host__ __device__ lambda cannot have internal or no linkage [C:\Jenkins\workspace\caffe2-builds\py2-cuda9.0-cudnn7-windows-build\build\caffe2\caffe2_gpu.vcxproj]
```